pkgs/os-specific/linux/mdio-netlink/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  kernel,
+  mdio-tools,
+}:
+
+stdenv.mkDerivation {
+  pname = "mdio-netlink";
+  version = mdio-tools.version;
```

Comment:
I know that this is more inconsistent than I'd like, but can you add `kernel.version` as a suffix, please?